kafka-connect-neo4j/src/main/kotlin/streams/kafka/connect/sink/Neo4jService.kt

```diff
@@ -42,20 +42,24 @@ class Neo4jService(private val config: Neo4jSinkConnectorConfig):
 
     init {
         val configBuilder = Config.builder()
-        if (this.config.encryptionEnabled) {
-            configBuilder.withEncryption()
-            val trustStrategy: Config.TrustStrategy = when (this.config.encryptionTrustStrategy) {
-                Config.TrustStrategy.Strategy.TRUST_ALL_CERTIFICATES -> Config.TrustStrategy.trustAllCertificates()
-                Config.TrustStrategy.Strategy.TRUST_SYSTEM_CA_SIGNED_CERTIFICATES -> Config.TrustStrategy.trustSystemCertificates()
-                Config.TrustStrategy.Strategy.TRUST_CUSTOM_CA_SIGNED_CERTIFICATES -> Config.TrustStrategy.trustCustomCertificateSignedBy(this.config.encryptionCACertificateFile)
-                else -> {
-                    throw ConfigException(Neo4jSinkConnectorConfig.ENCRYPTION_TRUST_STRATEGY, this.config.encryptionTrustStrategy.toString(), "Encryption Trust Strategy is not supported.")
+
+        if (!this.config.hasSecuredURI()) {
```

Comment:
I think this code is good and fine -- but setting explicit trust settings when using URLs like neo4j+s and neo4j+ssc is also not strictly needed.

Our drivers team has said that if you use neo4j+s or neo4j+ssc, then these settings are ignored.  If they conflict you could get an error (like for example using neo4j+s and also TRUST_ALL_CERTIFICATES, which doesnt' make sense) -- but if they don't conflict, it's not really needed.   Still this is OK